Tasks/DeployVisualStudioTestAgent/task.json

```diff
@@ -12,7 +12,7 @@
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 3
+        "Patch": 4
```

Comment:
this is a already changed in this sprint. please check and revert back this change